rotkehlchen/tests/api/test_statistics.py

```diff
@@ -650,7 +662,7 @@ def test_query_wrap(
         'BASE': 1,
     }
     assert result['top_days_by_number_of_transactions'] == [
-        {'timestamp': 1718496000, 'amount': '2'},
+        {'timestamp': 1734307200, 'amount': '4'},
```

Comment:
This number went from 2 to 4 because it is now counting based on `history_events` rather than `evm_transactions`, and the `history_events`/`evm_transactions` added in this test are not correlated the way they would be in real world data. I also added another EvmEvent using the same tx_hash as another event to confirm that it is still counting by transactions even though its not using the `evm_transactions` table.

Reply:
Did you add any ignored asset in this test?

Reply:
I've added a transaction and event with an ignored asset now. I also reworked the tx_hashes in the test so that the history_events correctly correspond to evm_transactions since the per chain tx count query now does a join on these two tables and needs this to be correct.